extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -184,9 +216,9 @@ public function refetchImageListByTimestamp( $timestamp ) {
 				'flags' 	=> $row->flags,
 				'wiki_url' 	=> isset( $wikiRow->city_url ) ? $wikiRow->city_url : '',
 				'user_page' => '', // @TODO fill this with url to user page
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)